2024-05-18 15:57:58,477 INFO    StreamThr :18968 [internal.py:wandb_internal():86] W&B internal server running at pid: 18968, started at: 2024-05-18 15:57:58.477001
2024-05-18 15:57:58,477 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: status
2024-05-18 15:57:58,477 INFO    WriterThread:18968 [datastore.py:open_for_write():85] open: c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155754-b1l3fxxt\run-b1l3fxxt.wandb
2024-05-18 15:57:58,478 DEBUG   SenderThread:18968 [sender.py:send():380] send: header
2024-05-18 15:57:58,478 DEBUG   SenderThread:18968 [sender.py:send():380] send: run
2024-05-18 15:57:58,842 INFO    SenderThread:18968 [dir_watcher.py:__init__():211] watching files in: c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155754-b1l3fxxt\files
2024-05-18 15:57:58,842 INFO    SenderThread:18968 [sender.py:_start_run_threads():1124] run started: b1l3fxxt with start time 1716040677.490268
2024-05-18 15:57:58,848 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: check_version
2024-05-18 15:57:58,848 DEBUG   SenderThread:18968 [sender.py:send_request():407] send_request: check_version
2024-05-18 15:57:58,922 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: run_start
2024-05-18 15:57:58,929 DEBUG   HandlerThread:18968 [system_info.py:__init__():32] System info init
2024-05-18 15:57:58,929 DEBUG   HandlerThread:18968 [system_info.py:__init__():47] System info init done
2024-05-18 15:57:58,929 INFO    HandlerThread:18968 [system_monitor.py:start():194] Starting system monitor
2024-05-18 15:57:58,929 INFO    SystemMonitor:18968 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-05-18 15:57:58,929 INFO    HandlerThread:18968 [system_monitor.py:probe():214] Collecting system info
2024-05-18 15:57:58,932 INFO    SystemMonitor:18968 [interfaces.py:start():190] Started cpu monitoring
2024-05-18 15:57:58,932 INFO    SystemMonitor:18968 [interfaces.py:start():190] Started disk monitoring
2024-05-18 15:57:58,935 DEBUG   HandlerThread:18968 [system_info.py:probe():196] Probing system
2024-05-18 15:57:58,935 INFO    SystemMonitor:18968 [interfaces.py:start():190] Started gpu monitoring
2024-05-18 15:57:58,939 INFO    SystemMonitor:18968 [interfaces.py:start():190] Started memory monitoring
2024-05-18 15:57:58,946 INFO    SystemMonitor:18968 [interfaces.py:start():190] Started network monitoring
2024-05-18 15:57:58,946 DEBUG   HandlerThread:18968 [system_info.py:_probe_git():181] Probing git
2024-05-18 15:57:58,984 DEBUG   HandlerThread:18968 [system_info.py:_probe_git():189] Probing git done
2024-05-18 15:57:58,984 DEBUG   HandlerThread:18968 [system_info.py:probe():244] Probing system done
2024-05-18 15:57:58,984 DEBUG   HandlerThread:18968 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22631-SP0', 'python': '3.10.11', 'heartbeatAt': '2024-05-18T13:57:58.935217', 'startedAt': '2024-05-18T13:57:54.435060', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'git': {'remote': 'https://github.com/aidos-lab/MANTRA.git', 'commit': 'b952d5d7ac2bf01ce1d8d6a60aefa9ac6fdd936d'}, 'email': 'ernstroell@gmail.com', 'root': 'C:/Users/ernst/Documents/02-Work/01-Projects/MANTRA', 'host': 'MININT-TDMRJO1', 'username': 'ernst', 'executable': 'C:\\Users\\ernst\\AppData\\Local\\Microsoft\\WindowsApps\\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\\python.exe', 'cpu_count': 16, 'cpu_count_logical': 24, 'cpu_freq': {'current': 3400.0, 'min': 0.0, 'max': 3400.0}, 'cpu_freq_per_core': [{'current': 3400.0, 'min': 0.0, 'max': 3400.0}], 'disk': {'/': {'total': 929.9218711853027, 'used': 827.0737457275391}}, 'gpu': 'NVIDIA GeForce RTX 4070', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4070', 'memory_total': 12878610432}], 'memory': {'total': 31.77265167236328}}
2024-05-18 15:57:58,984 INFO    HandlerThread:18968 [system_monitor.py:probe():224] Finished collecting system info
2024-05-18 15:57:58,984 INFO    HandlerThread:18968 [system_monitor.py:probe():227] Publishing system info
2024-05-18 15:57:58,984 DEBUG   HandlerThread:18968 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-05-18 15:57:58,985 DEBUG   HandlerThread:18968 [system_info.py:_save_pip():68] Saving pip packages done
2024-05-18 15:57:58,985 INFO    HandlerThread:18968 [system_monitor.py:probe():229] Finished publishing system info
2024-05-18 15:57:58,986 DEBUG   SenderThread:18968 [sender.py:send():380] send: files
2024-05-18 15:57:58,986 INFO    SenderThread:18968 [sender.py:_save_file():1380] saving file wandb-metadata.json with policy now
2024-05-18 15:57:58,993 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: stop_status
2024-05-18 15:57:58,994 DEBUG   SenderThread:18968 [sender.py:send_request():407] send_request: stop_status
2024-05-18 15:57:58,994 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 15:57:59,195 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: log_artifact
2024-05-18 15:57:59,206 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: partial_history
2024-05-18 15:57:59,206 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: pause
2024-05-18 15:57:59,206 INFO    HandlerThread:18968 [handler.py:handle_request_pause():705] stopping system metrics thread
2024-05-18 15:57:59,206 INFO    HandlerThread:18968 [system_monitor.py:finish():203] Stopping system monitor
2024-05-18 15:57:59,207 INFO    HandlerThread:18968 [interfaces.py:finish():202] Joined cpu monitor
2024-05-18 15:57:59,207 DEBUG   SystemMonitor:18968 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-05-18 15:57:59,211 INFO    HandlerThread:18968 [interfaces.py:finish():202] Joined disk monitor
2024-05-18 15:57:59,211 DEBUG   SystemMonitor:18968 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-05-18 15:57:59,211 DEBUG   SystemMonitor:18968 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-05-18 15:57:59,230 DEBUG   SenderThread:18968 [sender.py:send():380] send: telemetry
2024-05-18 15:57:59,230 DEBUG   SenderThread:18968 [sender.py:send_request():407] send_request: log_artifact
2024-05-18 15:57:59,240 INFO    HandlerThread:18968 [interfaces.py:finish():202] Joined gpu monitor
2024-05-18 15:57:59,240 INFO    HandlerThread:18968 [interfaces.py:finish():202] Joined memory monitor
2024-05-18 15:57:59,240 INFO    HandlerThread:18968 [interfaces.py:finish():202] Joined network monitor
2024-05-18 15:57:59,570 INFO    wandb-upload_0:18968 [upload_job.py:push():131] Uploaded file C:\Users\ernst\AppData\Local\Temp\tmpuob3474uwandb\2163tsbx-wandb-metadata.json
2024-05-18 15:57:59,848 INFO    Thread-84 :18968 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155754-b1l3fxxt\files\media\table\Table Name_0_2f4c7185e8b67feba4f1.table.json
2024-05-18 15:57:59,849 INFO    Thread-84 :18968 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155754-b1l3fxxt\files\wandb-metadata.json
2024-05-18 15:57:59,850 INFO    Thread-84 :18968 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155754-b1l3fxxt\files\requirements.txt
2024-05-18 15:57:59,851 INFO    Thread-84 :18968 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155754-b1l3fxxt\files\media
2024-05-18 15:57:59,851 INFO    Thread-84 :18968 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155754-b1l3fxxt\files\media\table
2024-05-18 15:58:00,262 INFO    wandb-upload_1:18968 [upload_job.py:push():131] Uploaded file C:\Users\ernst\AppData\Local\Temp\tmpuob3474uwandb\ccy4wur4-media/table/Table Name_0_2f4c7185e8b67feba4f1.table.json
2024-05-18 15:58:00,492 INFO    wandb-upload_0:18968 [upload_job.py:push():89] Uploaded file C:\Users\ernst\AppData\Local\wandb\wandb\artifacts\staging\tmp0i07bqup
2024-05-18 15:58:01,153 INFO    SenderThread:18968 [sender.py:send_request_log_artifact():1446] logged artifact run-b1l3fxxt-TableName - {'id': 'QXJ0aWZhY3Q6ODQxMjM5Nzk0', 'state': 'PENDING', 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjE3NjQwMDU5Mw==', 'latestArtifact': None}}
2024-05-18 15:58:01,153 DEBUG   SenderThread:18968 [sender.py:send():380] send: files
2024-05-18 15:58:01,153 INFO    SenderThread:18968 [sender.py:_save_file():1380] saving file media\table\Table Name_0_2f4c7185e8b67feba4f1.table.json with policy now
2024-05-18 15:58:01,154 DEBUG   SenderThread:18968 [sender.py:send():380] send: history
2024-05-18 15:58:01,154 DEBUG   SenderThread:18968 [sender.py:send_request():407] send_request: summary_record
2024-05-18 15:58:01,156 INFO    SenderThread:18968 [sender.py:_save_file():1380] saving file wandb-summary.json with policy end
2024-05-18 15:58:01,157 DEBUG   SenderThread:18968 [sender.py:send():380] send: stats
2024-05-18 15:58:01,635 INFO    wandb-upload_1:18968 [upload_job.py:push():131] Uploaded file C:\Users\ernst\AppData\Local\Temp\tmpuob3474uwandb\b0u8y1j1-media/table/Table Name_0_2f4c7185e8b67feba4f1.table.json
2024-05-18 15:58:01,876 INFO    Thread-84 :18968 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155754-b1l3fxxt\files\wandb-summary.json
2024-05-18 15:58:04,180 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: status_report
2024-05-18 15:58:09,249 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: status_report
2024-05-18 15:58:14,005 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: stop_status
2024-05-18 15:58:14,006 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 15:58:14,007 DEBUG   SenderThread:18968 [sender.py:send_request():407] send_request: stop_status
2024-05-18 15:58:15,223 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: status_report
2024-05-18 15:58:20,263 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: status_report
2024-05-18 15:58:25,317 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: status_report
2024-05-18 15:58:29,015 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: stop_status
2024-05-18 15:58:29,015 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 15:58:29,016 DEBUG   SenderThread:18968 [sender.py:send_request():407] send_request: stop_status
2024-05-18 15:58:31,231 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: status_report
2024-05-18 15:58:31,369 ERROR   SenderThread:18968 [internal_api.py:execute():370] 409 response executing GraphQL.
2024-05-18 15:58:31,369 ERROR   SenderThread:18968 [internal_api.py:execute():371] {"errors":[{"message":"run b1l3fxxt was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-05-18 15:58:32,667 ERROR   SenderThread:18968 [internal_api.py:execute():370] 409 response executing GraphQL.
2024-05-18 15:58:32,667 ERROR   SenderThread:18968 [internal_api.py:execute():371] {"errors":[{"message":"run b1l3fxxt was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-05-18 15:58:35,191 ERROR   SenderThread:18968 [internal_api.py:execute():370] 409 response executing GraphQL.
2024-05-18 15:58:35,192 ERROR   SenderThread:18968 [internal_api.py:execute():371] {"errors":[{"message":"run b1l3fxxt was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-05-18 15:58:35,193 INFO    SenderThread:18968 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\lib\retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\internal\internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\lib\gql_request.py", line 59, in execute
    request.raise_for_status()
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\requests\models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql
2024-05-18 15:58:39,995 ERROR   SenderThread:18968 [internal_api.py:execute():370] 409 response executing GraphQL.
2024-05-18 15:58:39,996 ERROR   SenderThread:18968 [internal_api.py:execute():371] {"errors":[{"message":"run b1l3fxxt was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-05-18 15:58:44,028 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 15:58:44,042 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: stop_status
2024-05-18 15:58:49,075 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 15:58:49,364 ERROR   SenderThread:18968 [internal_api.py:execute():370] 409 response executing GraphQL.
2024-05-18 15:58:49,364 ERROR   SenderThread:18968 [internal_api.py:execute():371] {"errors":[{"message":"run b1l3fxxt was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-05-18 15:58:54,105 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 15:58:59,028 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 15:59:04,189 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 15:59:09,075 ERROR   SenderThread:18968 [internal_api.py:execute():370] 409 response executing GraphQL.
2024-05-18 15:59:09,076 ERROR   SenderThread:18968 [internal_api.py:execute():371] {"errors":[{"message":"run b1l3fxxt was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-05-18 15:59:09,234 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 15:59:14,018 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 15:59:19,337 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 15:59:24,378 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 15:59:29,030 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 15:59:34,450 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 15:59:39,507 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 15:59:44,038 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 15:59:47,731 ERROR   SenderThread:18968 [internal_api.py:execute():370] 409 response executing GraphQL.
2024-05-18 15:59:47,731 ERROR   SenderThread:18968 [internal_api.py:execute():371] {"errors":[{"message":"run b1l3fxxt was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-05-18 15:59:49,587 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 15:59:54,615 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 15:59:59,048 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 16:00:04,697 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 16:00:09,740 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 16:00:14,058 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 16:00:19,830 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 16:00:24,879 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 16:00:29,069 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 16:00:34,996 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 16:00:40,037 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 16:00:44,072 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-18 16:00:49,126 DEBUG   HandlerThread:18968 [handler.py:handle_request():146] handle_request: keepalive
2024-05-18 16:00:53,354 ERROR   SenderThread:18968 [internal_api.py:execute():370] 409 response executing GraphQL.
2024-05-18 16:00:53,355 ERROR   SenderThread:18968 [internal_api.py:execute():371] {"errors":[{"message":"run b1l3fxxt was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-05-18 16:00:54,153 INFO    WriterThread:18968 [datastore.py:close():294] close: c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155754-b1l3fxxt\run-b1l3fxxt.wandb
2024-05-18 16:00:54,168 INFO    HandlerThread:18968 [handler.py:finish():866] shutting down handler
2024-05-18 16:00:54,168 ERROR   StreamThr :18968 [internal.py:wandb_internal():174] Thread SenderThread:
Traceback (most recent call last):
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\apis\normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\internal\internal_api.py", line 2008, in upsert_run
    response = self.gql(
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\internal\internal_api.py", line 338, in gql
    ret = self._retry_gql(
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\lib\retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\internal\internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\lib\gql_request.py", line 59, in execute
    request.raise_for_status()
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\requests\models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\internal\internal_util.py", line 49, in run
    self._run()
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\internal\internal_util.py", line 93, in _run
    self._debounce()
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\internal\internal.py", line 334, in _debounce
    self._sm.debounce()
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\internal\sender.py", line 538, in debounce
    self._maybe_update_config(always=final)
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\internal\sender.py", line 515, in _maybe_update_config
    self._debounce_config()
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\sdk\internal\sender.py", line 544, in _debounce_config
    self._api.upsert_run(
  File "C:\Users\ernst\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wandb\apis\normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: run b1l3fxxt was previously created and deleted; try a new run name (Error 409: Conflict)
